README.md

```diff
@@ -198,8 +198,39 @@ Every method call can throw `IncogniaHTTPError` and `IncogniaError`.
 
 ## How to Contribute
 
-If you have found a bug or if you have a feature request, please report them at this repository
-issues section.
+Your contributions are highly appreciated. If you have found a bug or if you have a feature request,
+please report them at this repository
+[issues section](https://github.com/inloco/incognia-python/issues).
+
+### Development
+
+#### Deployment
+
+On GitHub, you should merge your changes to the `main` branch, create the git
+[versioning](#Versioning) tag and finally push those changes:
+
+```shell
+$ git checkout main
+$ git pull
+$ git merge <your_branch>
+$ git tag -a v<version> -m <description> # the versioning tag requires the prefix v
```

Comment:
What you think about adding a filled example? The step is simple, but maybe it could be helpful